recipes/code-aster/meta.yaml

```diff
@@ -0,0 +1,111 @@
+{% set name = "code-aster" %}
+{% set version = "14.6" %}
+{% set minor_version = "11" %}
+{% set sha256 = "107ed98d10bc05866bdb3053f088fb0f51276ea14f7b615ec0d410a4b147ef12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - fn: {{ name }}_{{ version }}.{{ minor_version }}.tar.gz
+    url: https://gitlab.com/codeaster/src/-/archive/{{ version }}.{{ minor_version }}/src-{{ version }}.{{ minor_version }}.tar.gz
+    sha256: {{ sha256 }}
+  - url: https://downloads.sourceforge.net/project/prereq/metis-5.1.0_aster4.tar.gz
+    sha256: 1b295f9ce260c19d2d93fb2430be4139496c56f58f74ef7956ccd26c854fcc2e
+    folder: metis-aster
+  - url: http://graal.ens-lyon.fr/MUMPS/MUMPS_5.1.2.tar.gz
+    sha256: eb345cda145da9aea01b851d17e54e7eef08e16bfa148100ac1f7f046cd42ae9
+    folder: mumps-aster
+    patches:
+      - contrib/mumps-aster-diff.patch
+  - url: https://netcologne.dl.sourceforge.net/project/prereq/homard-11.12_aster2.tar.gz
+    sha256: 80425cee117349df8425f801f9a82aadb85fd41a95af25b4397f0e69dd655304
+    folder: homard
+    
+build:
+  number: 0
+  skip: true  # [py>=39 or osx or win or (python_impl == 'pypy')]
+  detect_binary_files_with_prefix: True
+  run_exports:
+    - gcc 8  # [linux]
+    - gxx 8  # [linux]
+    - gfortran 8  # [linux]
+  ignore_run_exports:
+    - numpy
+  run_exports:
+    - {{ pin_subpackage('code-aster', max_pin='x.x') }}
+    - {{ pin_compatible('python', max_pin='x.x') }}
+requirements:
+  build:
+    - gcc 8  # [linux]
+    - gxx 8  # [linux]
+    - gfortran 8  # [linux]
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - sysroot_linux-64 =2.17  # [linux]
+  host:
+    - cmake  
+    - make
+    - bison 
+    - flex 
+    - swig 
+    - python 3.8
+    - numpy {{ numpy }}
+    - mfront 3.2.1
+    - miss3d
+    - libmed 4.0
+    - hdf5 1.10.6
+    - scotch
+    - libblas
+    - liblapack       
+
+  run:
+    - python
+    - {{ pin_compatible('mfront', max_pin='x.x.x') }}
+    - {{ pin_compatible('miss3d', max_pin='x.x') }}
+    - {{ pin_compatible('numpy', max_pin='x.x') }}
+    - {{ pin_compatible('libmed', max_pin='x.x') }}
+    - {{ pin_compatible('hdf5', max_pin='x.x') }}
+    - libblas
+    - liblapack
+    - scotch
+    - libopenblas
+    - libgcc-ng  # [linux]
+    - libstdcxx-ng  # [linux]
+    - libgfortran5  # [linux]
+
+
+test:
+  imports:
+     - asrun
+#    - code_aster
+  commands:
+    - as_run --info 
+    - homard || true  # [unix]
+    - as_run --test forma01a
+    - as_run --test sslp114a
+    - as_run --test mumps02a
+    - as_run --test mumps01a
+    - as_run --test umat001a
+    - as_run --test zzzz121a
+    #- as_run --test mfron01a
+    - test -f $PREFIX/include/aster/aster.h
```

Comment:
There's lot of python scripts and a shared library being installed in `$PREFIX/lib64/aster`. We don't use `$PREFIX/lib64` because it is not added to the conda library path. Probably you want to move these to `$PREFIX/lib`.

Reply:
I think I fixed this one (I found the magic option and I do not see any lib64 in the output anymore :)